src/robot/libraries/String.py

```diff
@@ -346,6 +352,28 @@ def remove_string_using_regexp(self, string, *patterns):
         for pattern in patterns:
             string = self.replace_string_using_regexp(string, pattern, '')
         return string
+		
+    def remove_lines_containing_string(self, string, pattern, case_insensitive=False):
+        """Remove lines of the given ``string`` that contain the ``pattern``.
+
+        The ``pattern`` is always considered to be a normal string and a
+        line matches if the ``pattern`` is found anywhere in it. By
+        default the match is case-sensitive, but setting
+        ``case_insensitive`` to any value makes it case-insensitive.
+
+        Non matching lines are returned as one string catenated back together with
+        newlines. Possible trailing newline is never returned. The
+        number of non matching lines is automatically logged.
+
+		This does exact opposite functionality of Get Lines Containing String
```

Comment:
1. Weird indentation. 

2. Add backticks around referenced keyword names like

    `Get Lines Containing String`

  to automatically create a link to that keyword.

3. Add a note `New in Robot Framework 2.9.` at the end of the documentation.

4. You can review generated documentation by running

    doc/libraries/lib2html.py String